taccsite_cms/templates/guides/rsync_data_transfer.html

```diff
@@ -0,0 +1,510 @@
+      </p>
+      <h5 id="cy_windows">For Windows</h5>
+
+      <p>
+        <a target="_blank" href="https://cyberduck.io/download/">Download and install Cyberduck for Windows</a>
+        on your local machine.
+      </p>
+
+      <p>
+        Once installed, click “Open Connection” in the top left corner of your Cyberduck window.
+      </p>
+
+      <figure id="figure2">
+        <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/a9/9f/a99fa278-ae07-4b5e-9219-f9db3c2c89d5/picture2.png__1170x0_q85_subsampling-2_upscale.png" />
+        <figcaption>
+          Figure 2. Windows Cyberduck and “Open Connection” setup screen
+        </figcaption>
+      </figure>
+      <p>
+        To setup a connection, type in the server name
+        <kbd>host</kbd> (ex. <code>secure.corral.tacc.utexas.edu</code>). Add your TACC username and password in the spaces provided. If the "More Options" area is not shown, click the small triangle button to expand the window; this will allow you to enter the path to your transfer directory <kbd>/transfer/directory/path</kbd> (ex.
+        <code>/corral-secure/projects/A2CPS/submissions/tacc</code>) so that when Cyberduck opens the connection you will immediately be in your individualized transfer directory on the system. Click the "Connect" button to open your connection.
+      </p>
+
+      <p>
+        Consult Figure 3. below to ensure the information you have provided is correct.
+        <em>If you have not done so already, replace the “Path” with the path to your individualized transfer directory.</em>
+      </p>
+      <figure id="figure3">
+        <img src="https://a2cps-staging.tacc.utexas.edu/media/filer_public_thumbnails/filer_public/10/fb/10fbbb2a-dcd8-44d8-9ade-fb4ed3df8487/picture3.png__1170x0_q85_subsampling-2_upscale.png" />
+        <figcaption>
+          Figure 3. Windows “Open Connection” setup screen
+        </figcaption>
+      </figure>
+      <blockquote>
+        <p>
+          Note: You will be prompted to “allow unknown fingerprint…” upon connection. Select “allow” and enter your TACC token value.
+        </p>
+      </blockquote>
+      <p>
+        Once connected, you can navigate through your remote file hierarchy using familiar graphical navigation techniques. You may also drag-and-drop files from your local computer into the Cyberduck window to transfer files to the system.
+      </p>
+      <h5 id="cy_mac">For Mac</h5>
+
+      <p>
+        <a target="_blank" href="https://cyberduck.io/download/">Download and install Cyberduck for macOS</a>
+        on your local machine.
+      </p>
+
+      <p>
+        Once installed, go to Bookmark &gt; New Bookmark to setup a connection.
+      </p>
+      <blockquote>
+        <p>
+          Note: You cannot select “Open Connection” in the top left corner of your Cyberduck window as macOS’ setup screen is missing the “More Options” button.
+        </p>
+      </blockquote>
+      <p>
```

Comment:
SUGGESTION: Remove the example paths, they were established at the top of the documentation and their inclusion here makes it difficult to comprehend the intent of the section.